scanner/mappers/mappers.go

```diff
@@ -788,3 +788,64 @@ func findName(vuln *claircore.Vulnerability, p *regexp.Regexp) (string, bool) {
 	}
 	return "", false
 }
+
+// filterVulns filters repeat vulnerabilities out of vulnIDs and returns the result.
+// This function does not guarantee ordering is preserved.
+func filterVulns(vulnIDs []string, vulns map[string]*claircore.Vulnerability) []string {
```

Comment:
Nit:

```suggestion
func filterRepeatedVulns(vulnIDs []string, ccVulnerabilities map[string]*claircore.Vulnerability) []string {
```

- Make the function name specific.
- Keep the vulns map name the same, helps with readability.